appdatabase/migrations/sql/1721215212_create_keycard_and_accounts.up.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO keycards (keycard_uid, keycard_name, keycard_locked, key_uid, position)
+SELECT
+    s.keycard_instance_uid,
+    s.name,
+    0 AS keycard_locked,
+    s.key_uid,
+    0 AS position
+FROM settings s
+WHERE s.keycard_instance_uid IS NOT NULL
+  AND s.keycard_instance_uid NOT IN (SELECT keycard_uid FROM keycards);
+
+INSERT INTO keycards_accounts (keycard_uid, account_address)
+SELECT
+    k.keycard_uid,
+    kpa.address
+FROM keypairs_accounts kpa
+JOIN keycards k ON k.key_uid = kpa.key_uid
+WHERE kpa.wallet = 1
```

Comment:
I guess that because of this only the default wallet account (which belongs to the profile key pair) will be added to `keycards_accounts`table, while other accounts not. Shouldn't we add all wallet accounts for the profile keypair (those are all which have `chat == 0`?

Reply:
hm im not sure

Reply:
changed on chat = 0